kolibri/core/assets/src/views/ContentRenderer/filePresetStrings.js

```diff
@@ -0,0 +1,36 @@
+import findKey from 'lodash/findKey';
+import { createTranslator } from 'kolibri.utils.i18n';
+import bytesForHumans from 'kolibri.utils.bytesForHumans';
+
+// Strings are the _READABLE strings in le_utils.constants.format_presets,
+// with ' ({fileSize})' appended
+const filePresetStrings = {
+  highResolutionVideo: 'High Resolution ({fileSize})',
+  lowResolutionVideo: 'Low Resolution ({fileSize})',
+  vectorizedVideo: 'Vectorized ({fileSize})',
+  // Same 'thumbnail' string is used for video, audio, document, exercise, and topic
+  thumbnail: 'Thumbnail ({fileSize})',
+  videoSubtitle: 'Subtitle ({fileSize})',
+  audio: 'Audio ({fileSize})',
+  document: 'Document ({fileSize})',
+  exercise: 'Exercise ({fileSize})',
+  exerciseImage: 'Exercise Image ({fileSize})',
```

Comment:
We shouldn't need these three:
`exerciseImage`, `exerciseGraphie`, and `channelThumbnail`, as none of them will ever be shown to an end user for download.